packages/entities/entities-routes/vite.config.ts

```diff
@@ -20,22 +20,28 @@ const config = mergeConfig(sharedViteConfig, defineConfig({
     rollupOptions: {
       external: [
         '@kong-ui-public/expressions', // This is optional if we do not use Expressions features
+        '@kong-ui-public/expressions/dist/style.css', // This is optional if we do not use Expressions features
         'monaco-editor', // This is optional if we do not use Expressions features
       ],
     },
   },
-  server: {
-    proxy: {
-      // Add the API proxies to inject the Authorization header
-      ...getApiProxies(),
-    },
-  },
-  ...process.env.USE_SANDBOX && {
-    plugins: [
-      // See: https://github.com/vdesjs/vite-plugin-monaco-editor/issues/21
-      ((monacoEditorPlugin as any).default as typeof monacoEditorPlugin)({}),
-    ],
-  },
+  ...(process.env.USE_SANDBOX
+    ? {
+      server: {
+        proxy: {
+          // Add the API proxies to inject the Authorization header
+          ...getApiProxies(),
+        },
+      },
+    }
+    : {}),
+  plugins: [
+    // This plugin is only used in the sandbox & testing environment
+    // It generates extra files in dist folder whitch are not need in library build
+    ...((process.env.USE_SANDBOX || process.env.BABEL_ENV === 'cypress')
+      ? [((monacoEditorPlugin as any).default as typeof monacoEditorPlugin)({})]
+      : []),
+  ],
```

Comment:
issue: this plugins block can be removed from this root vite.config since it's in the package version